pages/about.js

```diff
@@ -7,11 +7,10 @@ const About = () => (
     <p className="title" >About</p>
     <div className="content">
       <div className="leftcolumn">
-        <p>
+        <div className="paragraph">
           The Code for America mission is, in part, to
           <blockquote>build open source technology and organize a network of people dedicated to making government services simple, effective, and easy to use.</blockquote>
-        </p>
-
+        </div>
```

Comment:
Instead of creating a new class and using a div, try just moving the `blockquote` outside of the `p` tag